### ISSUE TITLE
Ember 1.8 View attributeBindings do not set attribute if boolean (true) value

### ISSUE THREAD

**Issue by tomash** [emberjs/ember.js#9595]
In Ember 1.8 (tested with both 1.8.0 and 1.8.1) attributeBindings defined on a view **do not** add a given attribute to view's DOM element if the value is a boolean. In Ember 1.7 the same code has been setting a given attribute without any value, e.g. `<div data-modal>`.

Here's the jsfiddle demonstrating this issue (check out IndexView):
1.7.1 jsfiddle: http://jsfiddle.net/tomash/0w23rn6m/3/ (sometrueattribute set)
1.8.0 jsfiddle: http://jsfiddle.net/tomash/017d4Lmw/11/ (sometrueattribute not set)

If you change Ember in External Resources to 1.7, it works as expected.

Note: this **does not seem** to affect classNameBindings, they work as expected with boolean values.
Note 2: sometrueattribute on ApplicationView is **never** set in Ember 1.7 and 1.6, **unless** it's an (e.g.) tagName: 'input' and the boolean attribute is (e.g.) disabled: true.

This is my first issue reported in Emberjs -- I've read contributing guidelines but in case something's missing or wrong here, let me know.

**Comment by rwjblue** [comment]
This is finally fixed on canary (will be in 1.11.0-beta.1 shortly), thank you for your patience (and @mixonic 's hard work!!!)...

http://emberjs.jsbin.com/rwjblue/290/edit?html,js

**Comment by AdamFerguson** [comment]
@mixonic @rwjblue Thanks for the hard work!

**Comment by rlbaxter** [comment]
Upgrading our application a version at a time and ran into this as well when I got to 1.8 and 1.8.1. The bizarre thing is that it would sometimes work and sometimes wouldn't. For example, I have a custom select element view. I have required, disabled, and readonly as bound attributes. I would pass all three as true to the view, and the DOM element would get disabled and required, but not the readonly attribute.

Thanks for your hard work on this issue.
